app/views/application/_head.html.erb

```diff
@@ -1,9 +1,11 @@
 <meta charset='utf-8'>
 <%= tag :link, rel: "dns-prefetch", href: ENV["ASSET_HOST"] if ENV["ASSET_HOST"] %>
 <link rel="dns-prefetch" href="//www2.buildkiteassets.com" />
-<link rel="preload" as="font" type="font/woff2" href="https://www2.buildkiteassets.com/assets/lineto-circular/lineto-circular-book-39c5d1ed54e49102939d0280aeb20f01ef021bf5ffa74dc25fcafb43fce62ff3.woff2" crossorigin="anonymous" />
-<link rel="preload" as="font" type="font/woff2" href="https://www2.buildkiteassets.com/assets/lineto-circular/lineto-circular-bold-23e71bdfef13622d0d52b2b4b7ed3c1edb9e81f210692130dee9a521e97d062f.woff2" crossorigin="anonymous" />
-<link rel="preload" as="font" type="font/woff2" href="https://www2.buildkiteassets.com/assets/lineto-circular/lineto-circular-black-a8002849ea799a4dcf1be0b1abe0f010991cbae508f24f62e9ee0262590197eb.woff2" crossorigin="anonymous" >
+<link rel="preload" as="font" type="font/woff2" href="https://www2.buildkiteassets.com/assets/aeonik/Aeonik-Bold-78fff47500f27f2f0d8d777c9d431a87036e566af7577a58ccc0776b4eb4b994.woff2" crossorigin="anonymous" />
+<link rel="preload" as="font" type="font/woff2" href="https://www2.buildkiteassets.com/assets/maison-neue/MaisonNeue-Book-5e1404c0c8bc64daaae66d3a37fa7b0c7ffe65340d181e9068741deff8669cda.woff2" crossorigin="anonymous" />
+<link rel="preload" as="font" type="font/woff2" href="https://www2.buildkiteassets.com/assets/maison-neue/MaisonNeue-BookItalic-2a5e47b91af0050b44c21f9038ef63a37e228fd53b0289dfccbcdd3aa44ba19c.woff2" crossorigin="anonymous" />
```

Comment:
I think we should drop preloading these until we've landed the revised fonts. https://github.com/buildkite/buildkite/pull/11319

Reply:
You should be able to drop the font changes after a rebase.

Reply:
reverting back to Maison Bold, rather than Medium, because for Docs' use case the Medium font weight is too light. It makes sense for the visual design for Docs to diverge from Marketing Site here.